src/components/MapVisualization.css

```diff
@@ -1,9 +1,24 @@
+@media (max-width: 959px) {
+  .map-component {
+    display: none;
+  }
+}
+
+/*.map-component .leaflet-control-container {
+  width: 1600px;
+  overflow: hidden;
+  margin: 0 auto;
+  position: relative;
+  height: 300px;
+}*/
+
 .map-component .map-information {
+  /*display: none;*/
   border-bottom: 2px solid #ff6f00;
 }
 
 .map-component .map-information .info-container {
-  width: 100%;
+  width: 1536px;
```

Comment:
This number doesn't make sense.  Either explain or (preferably) don't hard code numbers in px.